Gemfile

```diff
@@ -39,7 +39,7 @@ gem 'hydra-role-management', '~> 0.2.0'
 gem 'rsolr', '~> 1.1.0'
 gem 'devise', '~> 3.0'
 gem 'devise-guests', '~> 0.3'
-gem 'iiif-presentation'
+gem 'iiif-presentation', github: 'tpendragon/osullivan', branch: 'no_singleton_classes'
```

Comment:
Want to merge this as-is, or wait for https://github.com/IIIF/osullivan/pull/58 to be merged?